tools/percolator/msgf2pin.py

```diff
@@ -0,0 +1,16 @@
+import sys
+
+from percolator import parse_options, run_percolator_command
+
+
+def main():
+    decoy = sys.argv[-1]
+    target = sys.argv[-2]
+    output = sys.argv[-3]
+    options = parse_options(sys.argv[1:-3])
+    command = ['msgf2pin', '-o', output]
```

Comment:
Can you not run this command directly in the XML file, without this additional python script?